integrations/ollama/tests/test_chat_generator.py

```diff
@@ -0,0 +1,159 @@
+from typing import List
+from unittest.mock import Mock
+
+import pytest
+from haystack.dataclasses import ChatMessage, ChatRole
+from requests import HTTPError, Response
+
+from ollama_haystack import OllamaChatGenerator
+
+
+@pytest.fixture
+def user_chat_message() -> ChatMessage:
+    msg = "Tell me about why Super Mario is the greatest superhero"
+    return ChatMessage.from_user(msg)
+
+
+@pytest.fixture
+def assistant_chat_message() -> ChatMessage:
+    msg = "Super Mario has prevented Bowser from destroying the world"
+    metadata = {"something": "something"}
+    return ChatMessage.from_assistant(msg, metadata)
+
+
+@pytest.fixture
+def list_of_chat_messages(user_chat_message, assistant_chat_message) -> List[ChatMessage]:
+    return [user_chat_message, assistant_chat_message]
```

Comment:
let's keep a unique fixture with a list of messages, please

Reply:
Related to https://github.com/deepset-ai/haystack-core-integrations/pull/176#discussion_r1444477501.

I would do something simpler, similar to https://github.com/deepset-ai/haystack/blob/ae96c2ee83ef06005885872780f8a976bc705677/test/components/generators/chat/test_openai.py#L11